app/controllers/LibAll.js

```diff
@@ -4,22 +4,14 @@
  * @author adrienjoly, whyd
  **/
 
-const mongodb = require('../models/mongodb.js');
 const postModel = require('../models/post.js');
 const feedTemplate = require('../templates/feed.js');
 
-function makeUserList() {
-  const userList = [];
-  for (const i in mongodb.usernames /*.slice(0,9)*/)
-    userList.push(mongodb.usernames[i]);
-  return userList;
-}
-
 function renderAllLibrary(lib) {
   const options = lib.options;
   //options.displayAuthors = true;
   options.displayPlaylistName = true;
-  options.follows = { people: makeUserList(), followers: [] };
+  options.follows = { people: [], followers: [] }; // TODO: fix the /all page without having to fetch all users
```

Comment:
Note: the `/all` page is probably broken because of this, but the DB cost would be to great otherwise.